src/core/server/saved_objects/saved_objects_service.ts

```diff
@@ -475,14 +475,9 @@ export class SavedObjectsService
 }
 
 function getImportableAndExportableTypes({
-  savedObjectMappings = [],
   savedObjectsManagement = {},
 }: SavedObjectsLegacyUiExports) {
-  const visibleTypes = savedObjectMappings.reduce(
```

Comment:
@pgayvallet Could you please verify that change? It looks like some extra check but after moving registering  of '`index-pattern`' type into `data` plugin `import/export` functionality doesn't work.  

Also please keep in mind that we should be able to register more types from the '`setup`' hook of our custom plugin (e.g. data plugin), and this call may be executing after the 'setup' hook of the '`saved-objects`' plugin.

Reply:
You are right, types registered from NP plugins are currently excluded from this list, therefor considered as not exportable. I kinda missed that in the initial implementation.

Good new is, this is addressed by https://github.com/elastic/kibana/pull/59291 as you will be able to register the `management` section using the same `registerType` API.

Keep in mind that you will need to add the `management` section to your types registration ( using both `registerType` without `management` from NP and the associated `uiExports.savedObjectsManagement` from legacy plugin will not connect the two)

Reply:
@pgayvallet does it mean that I should revert this part and wait until #59291 will be merged?  Or it's ok for now